src/views/poi_bloc/Wiki.jsx

```diff
@@ -0,0 +1,25 @@
+/* global _ */
+import React from 'react';
+import ExtendedString from '../../libs/string';
+
+const WikiBlock = ({
+  block,
+}) => {
+  return <div className="poi_panel__info__wiki">
+    { block.description &&
+      <p className="poi_panel__description__ellipsis">
+        { block.description }
+      </p> }
+    <br/>
+    { block.url &&
+      <a
+        className="poi_panel__info__wiki__link"
+        rel="noopener noreferrer"
+        target="_blank"
+        href={ ExtendedString.htmlEncode(block.url) } >
```

Comment:
I'm sure this one is not needed, especially here for a url. It will just make a mess if it includes `&` character, resulting in a wrong url.